pages/submitted-expenses.tsx

```diff
@@ -0,0 +1,366 @@
+      pathname: `${getCollectivePageCanonicalURL(collective)}/submitted-expenses`,
+      query: this.buildFilterLinkParams({ ...queryParams, offset: null }),
+    });
+  };
+
+  handleSearch = (searchTerm, collective) => {
+    const params = this.buildFilterLinkParams({ searchTerm, offset: null });
+    this.props.router.push({
+      pathname: `${getCollectivePageCanonicalURL(collective)}/submitted-expenses`,
+      query: params,
+    });
+  };
+
+  getTagProps = tag => {
+    if (tag === this.props.query.tag) {
+      return { type: 'info', closeButtonProps: true };
+    }
+  };
+
+  getShouldDisplayFeatureNotSupported = account => {
+    if (!account) {
+      return true;
+    }
+  };
+
+  getSuggestedTags = memoizeOne(getSuggestedTags);
+
+  render() {
+    const { collectiveSlug, data, query } = this.props;
+    const hasFilters = this.hasFilter(query);
+    const pageUrl = `${getCollectivePageCanonicalURL(data.account)}/submitted-expenses`;
+
+    if (!data.loading) {
+      if (data.error) {
+        return <ErrorPage data={data} />;
+      } else if (!data.account || !data.expenses?.nodes) {
+        return <ErrorPage error={generateNotFoundError(collectiveSlug)} log={false} />;
+      } else if (!isIndividualAccount(data.account)) {
+        // Hack for funds that want to keep their budget "private"
+        return <PageFeatureNotSupported showContactSupportLink={false} />;
+      }
+    }
+
+    return (
+      <Page collective={data.account} canonicalURL={pageUrl} {...this.getPageMetaData(data.account)}>
+        <CollectiveNavbar
+          collective={data.account}
+          isLoading={!data.account}
+          selectedCategory={NAVBAR_CATEGORIES.BUDGET}
+        />
+        <Container position="relative" minHeight={[null, 800]}>
+          <Box maxWidth={Dimensions.MAX_SECTION_WIDTH} m="0 auto" px={[2, 3, 4]} py={[0, 5]} mt={3}>
+            <Flex justifyContent="space-between" flexWrap="wrap">
+              <Box flex="1 1 500px" minWidth={300} maxWidth={'100%'} mr={0} mb={5}>
+                <Flex>
+                  <H1 fontSize="32px" lineHeight="40px" py={2} fontWeight="normal">
+                    {data.account && isIndividualAccount(data.account) ? (
+                      <FormattedMessage defaultMessage="Submitted Expenses" />
+                    ) : (
+                      <FormattedMessage id="Expenses" defaultMessage="Expenses" />
+                    )}
+                  </H1>
+                  <Box mx="auto" />
+                  <SearchFormContainer p={2}>
+                    <SearchBar
+                      defaultValue={query.searchTerm}
+                      onSubmit={searchTerm => this.handleSearch(searchTerm, data.account)}
+                    />
+                  </SearchFormContainer>
+                </Flex>
+                <StyledHr my={24} mx="8px" borderWidth="0.5px" />
+                <Box mx="8px">
+                  {data.account ? (
+                    <ExpensesFilters
```

Comment:
We need to make sure we add a property to ignore `IGNORED_EXPENSE_STATUS` inside `ExpensesStatusFilter`, otherwise, we'll never display _Draft_ as an option to filter.